dataactcore/models/errorInterface.py

```diff
@@ -57,14 +57,7 @@ def getFileStatusLabelByJobId(self, jobId):
         return self.runUniqueQuery(query,"No file for that job ID", "Multiple files have been associated with that job ID").file_status.name
 
     def checkNumberOfErrorsByJobId(self, jobId, valDb, errorType = "fatal"):
-        """ Get the total number of errors for a specified job
-
-        Args:
-            jobId: job to get errors for
-
-        Returns:
-            Number of errors for specified job
-        """
+        """Deprecated: moved to function_bag.py."""
```

Comment:
Hmmm, tests failed all over the place when I did this, with sqlalchemy/postgres errors. I'm not inclined to spend time debugging when it's a temporary situation so I'd prefer to keep the duplicated code/deprecated message and revisit when doing the larger task of moving interface code into the function_bag.